brand/admin.py

```diff
@@ -3,6 +3,7 @@
 from django.contrib.admin.widgets import FilteredSelectMultiple
 from django.urls import reverse
 from django.utils.html import escape, format_html
+from django_object_actions import DjangoObjectActions
```

Comment:
I'm glad the library is well supported but it seems a bit unnecedssary to use django_object_actions here. Is there a way to do this without introducing a new dependency?